frontend/src/metabase/ui/theme.ts

```diff
@@ -95,6 +95,7 @@ export const getThemeOverrides = (): MantineThemeOverride => ({
       },
     },
   },
+  fontFamily: "var(--mb-default-font-family)",
```

Comment:
I think we generally follow up the font family with `sans-serif`. Should we do that here too?
![image](https://github.com/metabase/metabase/assets/1328979/12403530-964f-41a8-8027-97f1cf329f91)

Reply:
good idea! this will at least get me arial instead of times when my site url is incorrect